source/documentation/gluster-hyperconverged/chap-Deploying_Hyperconverged.html.md

```diff
@@ -33,6 +33,10 @@ title: Deploying oVirt and Gluster Hyperconverged
 
         # yum install ovirt-engine-appliance
 
+4. Install the Gluster Plugin for VDSM.
+
+        # yum install vdsm-gluster     
```

Comment:
You can make this part of step 2 itself. Edit the description to include vdsm-gluster (plugin to manage gluster services).
While you're at it, can you also change the section to say Install on all 3 hosts. 

Step 3 - is only needed on the first host.